### ISSUE TITLE
android 11 (custom rom): wrong location for forward_ip

### ISSUE THREAD

**Issue by CircleCode** [IngoZenz/personaldnsfilter#138]
I'm using android 11 with msm extended rom (https://msmxtended.org/) on oneplus3T
Magisk is installed, and PersonaDNSFilter has root permissions
while trying to enable root mode without local vpn, iptable commands run sucessfully, but then comes the following error: `/forward_ip: open failed: EROFS (Read-only file system)`
It appears that the `forward_ip` script is created at the wrong location (at `/`) and this is indeed not allowed

**Comment by IngoZenz** [comment]
This was a known issue reported for some devices, but just understood now...
Should be fixed in dev with https://github.com/IngoZenz/personaldnsfilter/commit/bcdd2cf9da29e91ebcfd023d733f64e7003220e0

In case you would like the fix from dev please find build attached. It should be stable. Feedback welcome!
[DNSFilter-1.5047-dev-0.zip](https://github.com/IngoZenz/personaldnsfilter/files/5834800/DNSFilter-1.5047-dev-0.zip)

**Comment by CircleCode** [comment]
Thanks for the quick fix.
Are you confident in this fix, or do you need me to test it?
I'm asking because I installed pDNSf from fdroid, and testing this version requires me to uninstall f-droid version, and to uninstall this version once the update is available through f-droid.
However, if you need a test, I'll happily do it

**Comment by IngoZenz** [comment]
I am pretty confident but I do not have a rooted phone at hand currently to test.

Btw: Uninstall will not remove the config - it will stay.

But in case you do not want to check this - I can understand and it is fine as well :-)

**Comment by CircleCode** [comment]
I tried and this problem is solved. however, it seems there is another problem, and I do not know if this is related to this release:
I configured the following dns server: `[193.70.85.11]::443::DOH::doh.bortzmeyer.fr`, and the logs give
```
DoH failed for doh.bortzmeyer.fr! 404 - HTTP/1.1 404 Not Found
IO Error occured! Check network or dDNS config!
```
I tried this server from a computer, and it seems to work correctly.
But I do not know how to debug the ongoing problem on android.
